entrypoint.sh

```diff
@@ -20,6 +20,7 @@ if [ "$1" = "kube-proxy" ]; then
 fi
 
 if [ "$1" = "kubelet" ]; then
+    CGROUPDRIVER=$(/opt/rke-tools/bin/docker info | grep -i 'cgroup driver' | awk '{print $3}')
     DOCKER_ROOT=$(DOCKER_API_VERSION=1.24 /opt/rke-tools/bin/docker info 2>&1  | grep -i 'docker root dir' | cut -f2 -d:)
```

Comment:
```suggestion
    CGROUPDRIVER=$(/opt/rke-tools/bin/docker info -f '{{.Info.CgroupDriver}}')
    DOCKER_ROOT=$(DOCKER_API_VERSION=1.24 /opt/rke-tools/bin/docker info -f '{{.Info.DockerRootDir}}')
```

Reply:
Thanks for the suggestion @brandond 
The `-f` option surely is more effective but unfortunately we can't use it as the docker binary shipped in the rke-tools image is quite old (v1.12.3): https://github.com/rancher/rke-tools/blob/master/package/Dockerfile#L5

Updating the docker binary could be a good thing but that would probably need a separate PR

Reply:
Docker 1.12.3 is like 7 years old! It was released 2016-10-26 and is undoubtedly riddled with vulnerabilities. Now is probably a good time to update that - perhaps in another commit in this same PR?

cc @macedogm - if we're not scanning the rke-tools stuff for vulns, we should start doing so immediately. I see a bunch of very old versions in use in this repo.

Reply:
also cc @snasovich on this thread

Reply:
Hi @brandond, we are scanning rke-tools and also aware of this. We've an internal issue open to track this, as well as a big PR https://github.com/rancher/rke-tools/pull/161 to bump a bunch of dependencies.

Hi @snasovich, could we get a second colleague from team/area2 to help review the PR and perhaps merge it, please?

CC @pjbgf

Reply:
FYI: the Docker binary has now been bumped to `20.10.24` on [rke-tools:v0.1.91](https://github.com/rancher/rke-tools/releases/tag/v0.1.91).